src/scripts/elasticsearch-ubuntu-install.sh

```diff
@@ -182,6 +182,9 @@ while getopts :n:m:v:A:R:K:S:Z:p:a:k:L:C:B:Xxyzldjh optname; do
     k) #azure storage account key for azure cloud plugin
       STORAGE_KEY="${OPTARG}"
       ;;
+    E) #azure storage account endpoint suffix
```

Comment:
Want to keep the provisioning of a storage account for azure-repository plugin separate.